addons/website/static/src/components/fields/fields.xml

```diff
@@ -56,9 +56,6 @@
             t-model="state.name"
             t-ref="input"
         />
-        <div t-if="warnAboutCall" class="mt-4">
-            <PageDependencies pageId="props.record.resId" mode="'popover'" type="'key'"/>
-        </div>
```

Comment:
Should not the whole template `website.PageNameField` be removed by this first commit? :thinking: